cli/flox/src/commands/activate.rs

```diff
@@ -321,7 +330,8 @@ impl Activate {
     fn launch_watchdog(
         flox: &Flox,
         cache_path: PathBuf,
-        vars: &HashMap<&str, String>,
+        path_hash: &str,
+        socket_path: impl AsRef<Path>,
     ) -> Result<()> {
         let mut cmd = Command::new(&*KLAUS_BIN);
 
```

Comment:
is there a reason to unit test this, or too low value/effort ratio?

Reply:
My gut feeling is that the logic is simple enough to be low value and that inspecting the effects would be high effort.

Reply:
I'm not sure what you would really be testing. It's basically just setting arguments and spawning a process. The interesting part is what the watchdog actually does.